samples/BasicXrApp/BasicXrApp_win32.vcxproj

```diff
@@ -70,7 +68,7 @@
     <Link>
       <SubSystem>Windows</SubSystem>
```

Comment:
We can remove the "/permissive- /bigobj" as well as 4447 warning from this project as well.